pkg/reconciler/image/reconcile_build.go

```diff
@@ -35,6 +35,7 @@ func (c *Reconciler) reconcileBuild(image *v1alpha1.Image, latestBuild *v1alpha1
 			BuildCounter:               nextBuildNumber,
 			BuildCacheName:             buildCacheName,
 			LatestBuildRef:             build.BuildRef(),
+			LatestBuildReason:          build.BuildReason(),
```

Comment:
I think we need the LatestBuildReason populated when a build is not scheduled. (ConditionUnknown and ConditionFalse cases)

Reply:
Got it -- I'll add it to the ConditionFalse case (looks like ConditionUnknown is just a fallthrough). Thanks!

Reply:
@matthewmcnew - I added the LatestBuildReason to the ConditionFalse case in the newest commit.